ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/QueryEndpoint.java

```diff
@@ -173,6 +190,14 @@ private static List<String> colNamesFromSchema(final List<Column> columns) {
         .collect(Collectors.toList());
   }
 
+  private Map<String, Object> consumerProperties(Map<String, Object> properties) {
```

Comment:
nit: this logic feels like it belongs with BlockingPrintPublisher (or one of the other classes specific to PRINT TOPIC), rather than the QueryEndpoint.

Reply:
I agree, moving this to BlockingPrintPublisher.